pkg/endpoint/endpoint.go

```diff
@@ -807,6 +815,16 @@ func (e *Endpoint) GetLabels() []string {
 	return e.SecurityIdentity.Labels.GetModel()
 }
 
+// GetSecurityIdentity returns the security identity of the endpoint.
+func (e *Endpoint) GetSecurityIdentity() *identityPkg.Identity {
+	return e.SecurityIdentity
+}
+
+// GetID16 returns the endpoint's ID as a 16-bit unsigned integer.
+func (e *Endpoint) GetID16() uint16 {
```

Comment:
(for later) We should really create an EndpointID type that handles this nicely everywhere...